common/app/views/fragments/nav/userAccountLinks.scala.html

```diff
@@ -14,49 +13,47 @@
     </li>
 
     <li class="menu-item u-h js-navigation-account-details">
-        <details>
-            <summary class="menu-item__title"
-                     data-link-name="nav2 : my account">
-                <i class="menu-item__toggle"></i>
-                my account
-            </summary>
+        <button class="menu-item__title js-navigation-toggle"
```

Comment:
Well without JS we will just always show the `sign in` link here anyway won't we? No one without JS will ever see the `my account` at all. I don't think it makes sense to be a link if thats true.